sympy/sets/sets.py

```diff
@@ -980,6 +980,82 @@ def _contains(self, other):
 
         return _sympify(expr)
 
+    def _add(self, other):
+        """
+        Additions in interval arithmetic
+        https://en.wikipedia.org/wiki/Interval_arithmetic
+        """
+        if isinstance(other, Interval):
+            return Interval(self.start + other.start, self.end + other.end, self.left_open | other.left_open, self.right_open | other.right_open)
+        raise NotImplementedError()
+
+    def _sub(self, other):
+        """
+        Subtractions in interval arithmetic
+        https://en.wikipedia.org/wiki/Interval_arithmetic
+        """
+        if isinstance(other, Interval):
+            return Interval(self.start - other.end, self.end - other.start, self.left_open | other.right_open, self.right_open | other.left_open)
```

Comment:
`or` is better, unless you explicitly want to create a symbolic object. It's more readable, and can't accidentally do bitwise or.